qa/workunits/rgw/test_librgw_file.sh

```diff
@@ -6,7 +6,7 @@ then
     export AWS_ACCESS_KEY_ID=`openssl rand -base64 20`
     export AWS_SECRET_ACCESS_KEY=`openssl rand -base64 40`
 
-    radosgw-admin user create --uid ceph-test-librgw-file \
+    radosgw-admin ${CEPH_CONF} user create --uid ceph-test-librgw-file \
```

Comment:
ceph already supports the `CEPH_CONF` environment variable (see `getenv("CEPH_CONF")` in src/common/config.cc)

i would expect this script to work without changes, if you invoke it with `CEPH_CONF=/path/to/ceph.conf`

Reply:
Interesting.  I believe it didn't, but I'll try again, and if it does, remove this commit.

Reply:
I must have done something wrong, it works fine.

Reply:
\o/